src/main/java/Duke.java

```diff
@@ -74,12 +78,20 @@ public static void printList(Task[] args, int length) {
         System.out.println("Here are the tasks in your list:");
         for (int i = 0; i < length; i++) {
             System.out.print(i + 1 + ".");
-            System.out.print("[" + args[i].getStatusIcon() + "]");
-            System.out.println(" " + args[i].description);
+            System.out.println(args[i].toString());
         }
         printSign();
     }
 
+    public static void printTotalNumOfTasks(Task[] args, int total) {
```

Comment:
I find this practice very good. Create a function specifically to print out the number of tasks